frontend/src/pages/DashboardPage/Dashboard/Dashboard.jsx

```diff
@@ -419,7 +476,7 @@ const Dashboard = () => {
       })
       .then((response) => {
         console.log("Data generated:", response.data); // TODO Display data on chart
-        setGraphData(response.data);
+        // setGraphData(response.data);
```

Comment:
Can you please uncomment this, as we need the data to be displayed

Reply:
Sure! Once we implement the API calls and fetch the necessary data dynamically instead of using hardcoded values, we can uncomment this section. This will allow the data to be displayed properly. Let me know if you'd like me to assist with the API integration!